packages/web-react/src/components/TextArea/README.md

```diff
@@ -1,11 +1,11 @@
 # TextArea
 
 TextArea enables the user to add longer text to a form. It has textarea, label,
-and an optional message. It could be disabled or have a validation state. The label could be hidden
+and an optional helper text. It could be disabled or have a validation state. The label could be hidden
```

Comment:
```suggestion
and an optional validation text. It could be disabled or have a validation state. The label could be hidden
```

Reply:
I don't think so. IMHO we should here mention helper text and we can mention validation text next to the validation state, because it is dependent on it.